frontend/src/types/LogRecordTypes.ts

```diff
@@ -1,8 +1,8 @@
 export type LogRecord = {
   logId: number;
-  attnTo: number;
-  attnToFirstName: string;
-  attnToLastName: string;
```

Comment:
Going to add a ticket for this as well - better to have it structured like:
`attnTo: AttnToType`

`type AttnTo = {
id: ...
firstName...
lastName...
}`

Think the same thing can be done for employee here - having it optional is correct though